tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -268,6 +276,12 @@ install_initrd() {
 	"${rootfs_builder}" --osname="${os_name}" --osversion="${os_version}" --imagetype=initrd --prefix="${prefix}" --destdir="${destdir}" --image_initrd_suffix="${variant}"
 }
 
+#Install guest initrd for sev
```

Comment:
"sev" should be "confidential guests" maybe?

Reply:
Fixed, thanks!